app/views/ExposureHistory/MoreInfo.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import { View, ScrollView, StyleSheet } from 'react-native';
+import { useNavigation } from '@react-navigation/native';
+import { useTranslation } from 'react-i18next';
+
+import { Typography } from '../../components/Typography';
+import { NavigationBarWrapper } from '../../components/NavigationBarWrapper';
+import { useStatusBarEffect } from '../../navigation';
+
+import { Spacing, Typography as TypographyStyles } from '../../styles';
+
+const MoreInfo = (): JSX.Element => {
+  const { t } = useTranslation();
+  const navigation = useNavigation();
+  useStatusBarEffect('light-content');
+
+  const handleOnBackPress = () => {
+    navigation.goBack();
+  };
+
+  const headerTextOne = t('exposure_history.why_did_i_get_an_en');
```

Comment:
@JacobJaffe , this is one where there might be some differences between gps and bt, what are your thoughts on this? is the copy generic enough or should we rope in product?